### PR TITLE
stackrox: finally onboard

### PR DESCRIPTION
**Pull request by gavin-stackrox**
This PR onboards https://github.com/stackrox/stackrox to OpenShift CI with an initial lightweight: `style-checks`.

### CODE REVIEW

**Comment by gavin-stackrox** [comment]
A note to reviewers: Prow through its bunderbuss plugin autoassigns reviewers. IIRC this can be avoided by choosing 2 reviewers in the initial PR description.
/uncc @msugakov 
/cc @JoukoVirtanen

**Review by sbostick — APPROVED** [review]
LGTM

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: stackrox-test
+    namespace: stackrox
+    tag: latest
+releases:
+  latest:
+    integration:
+      name: "4.8"
```
Comment (vjwilson) [inline ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:14]: Isn't OpenShift 4.10 the latest GA release?

File: core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+    plugins:
+    - assign
+    - cat
+    - dog
+    - goose
+    - heart
+    - help
+    - label
+    - override
+    - pony
```
Comment (vjwilson) [inline core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml:27]: Why the menagerie?

**Review by vjwilson — APPROVED** [review]
/lgtm

File: ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml (hunk 1 above)
Comment (gavin-stackrox) [reply ci-operator/config/stackrox/stackrox/stackrox-stackrox-master.yaml:14]: Don't know and tbh I don't know how this stanza affects CI but it is required, so I copied what is used in other places.

File: core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml (hunk 2 above)
Comment (gavin-stackrox) [reply core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml:27]: /goose

Comment (gavin-stackrox) [reply core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml:27]: /dog

Comment (gavin-stackrox) [reply core-services/prow/02_config/stackrox/stackrox/_pluginconfig.yaml:27]: /woof

**Pull request merged**